modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/components/SearchHeader/components/SearchBar/SearchBar.test.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {faker} from '@faker-js/faker';
+import {fireEvent, render} from '@testing-library/react';
+import i18n from '../../../../../../common/I18n';
+import SearchBar from './';
+
+describe('render Search bar on Overview page', () => {
```

Comment:
We could change the test suit description to specify the component itself, like `Overview Page: SearchBar`, what do you think?

```suggestion
describe('Overview Page: SearchBar', () => {
```

Reply:
Inside describe, usually, we put the Component name, or at least describe what the test suite will do, for example

```suggestion
describe('SearchBar', () => {
```

or

```suggestion
describe('Test SearchBar Using blábláblá', () => {
```

The text `render Search bar on Overview page` makes sense, but applied to `it` or `test` inside the describe callback.

```js
describe('SearchBar', () => {
     it('render Search bar on Overview page', () => {})
})
```